plot_types/A_basic/a_plot.py

```diff
@@ -0,0 +1,23 @@
+"""
+======================
+plot([X], Y, [fmt]...)
+======================
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+# make data
+X = np.linspace(0, 10, 100)
+Y = 4 + 2 * np.sin(2 * X)
+
+# plot
+with plt.style.context('cheatsheet_gallery'):
+    fig, ax = plt.subplots()
+
+    ax.plot(X, Y, color="C1", linewidth=2.0)
```

Comment:
Any reason for using C1 (orange)?  just from the peanut gallery, C0 (blue) may perhaps look better, and it's the default too?

Reply:
OK, so I tried changing the color order.  Guess what - a bunch of artists do not respect the the rcParam when changed in the context manager.  i.e. most of the stats ones.  

WRT to orange, that is for consistency with the cheatsheet.  You can argue with @rougier about what looks better ;-)

Reply:
OK, the issue with the colororder not sticking was if `plt.show()` was inside the context manager or outside.  Some artists delay assigning the color until draw time.  I think we should stomp out any delayed state in the library as it almost always makes things confusing....

Reply:
I think "C0" might do, it mostly depends on the matplotlib website main color.